cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,30 @@
+class PaymentStepPage {
+    private BtBankWire: string;
+    private CartNavigation: string
+    private CenterColumn: string
+
+    constructor() {
+        this.BtBankWire = '.bankwire';
+        this.CartNavigation = '.cart_navigation span'
```

Comment:
This is referring to the proceed to checkout button, let's name this property in that way.